### PR TITLE
speedtest-rs: add page

### PR DESCRIPTION
**Pull request by Abhishek-1804**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page(s) follow the [style guide](/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Comment by Abhishek-1804** [comment]
@sebastiaanspeck @acuteenvy Hi, Could you please review this pr?

File: pages/common/speedtest-rs.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# speedtest-rs
+
+> A Speedtest.net testing utility and crate for checking network download and upload speeds.
```
Comment (gutjuri) [inline pages/common/speedtest-rs.md:3]:
```suggestion
> A `speedtest.net` testing utility and crate for checking network download and upload speeds.
```

File: pages/common/speedtest-rs.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+> More information: <https://github.com/nelsonjchen/speedtest-rs>.
+
+- Run a full speed test (download and upload):
+
+`speedtest-rs`
+
+- Display a list of speedtest.net servers sorted by distance:
```
Comment (gutjuri) [inline pages/common/speedtest-rs.md:10]:
```suggestion
- Display a list of `speedtest.net` servers sorted by distance:
```

**Review by gutjuri — APPROVED** [review]
Thanks! I have two minor suggestions.

**Comment by Managor** [comment]
`speedtest` and `speedtest-rs` seem to have indentical syntax. These pages should be merged and speedtest-rs be made into an alias.

**Comment by Managor** [comment]
#14804

**Comment by Abhishek-1804** [comment]
> `speedtest` and `speedtest-rs` seem to have indentical syntax. These pages should be merged and `speedtest-rs` be made into an alias.

While speedtest-rs and speedtest-cli might seem similar, they’re not aliases. speedtest-rs is written in Rust and was created as a learning project, focusing on HTTP Legacy Fallback. It’s missing features like TCP-based connections and Speedtest Mini support, which speedtest-cli includes. These differences make them separate tools with distinct implementations.

**Comment by Managor** [comment]
I see, thank you. Could you be able to modify the page for `speedtest` so that these pages are more uniform? According to that issue, the examples are not functional.

**Comment by Abhishek-1804** [comment]
I understand the confusion now. I’ll update the documentation to make the distinction between speedtest, speedtest-cli, and speedtest-rs clearer, especially with regard to different systems where speedtest might point to non- Ookla tools like the GUI-based librespeed

**Review by Managor — APPROVED** [review]

**Pull request merged**